src/MCPClient/lib/clientScripts/copySubmissionDocs.sh

```diff
@@ -8,3 +8,4 @@
 subdir="$1/submissionDocumentation"
 mkdir -p $subdir
 cp -R "$1/$2/data/objects/submissionDocumentation" $subdir
```

Comment:
(The reason I wonder about that specifically is because I'm interested at some point in adopting the `set -e` behaviour [described here](http://redsymbol.net/articles/unofficial-bash-strict-mode/) - so all non-zero exit commands fail the script even if they're not the final line.)